### ISSUE TITLE
GitHub Workflows can probably do this now.

### ISSUE THREAD

**Issue by Lupus590** [SquidDev-CC/howl.ci#12]

**Comment by SquidDev** [comment]
It's probably possible (https://docs.github.com/en/rest/actions/workflow-runs#download-workflow-run-attempt-logs), but I'm afraid I'm not going to implement this. I'm not really using howl.ci any more.

**Comment by Lupus590** [comment]
Yeah, this is more of an idea thrown out there for anyone to have a go at.
